WordPress/src/main/java/org/wordpress/android/ui/mysite/SiteIconUploadHandler.kt

```diff
@@ -74,8 +74,6 @@ class SiteIconUploadHandler
                 if (event.mediaModelList.size > 0) {
                     val media = event.mediaModelList[0]
                     selectedSiteRepository.updateSiteIconMediaId(media.mediaId.toInt(), true)
-                    selectedSiteRepository.showSiteIconProgressBar(false)
```

Comment:
These lines are not needed anymore. Not sure if we need `selectedSiteRepository.refresh()` function anymore, but I didn't want to touch it, since I have no context.

Reply:
To be honest, I am not sure as well Pantelis, but to avoid any regression issues we can ignore this for now. Selected site repository needs to be taken up and changed in the future anyways so we can re-visit then may be.